src/database/DatabaseBackend.h

```diff
@@ -2,8 +2,30 @@
 #include "DBOperation.h"
 #include "config/ConfigVariable.h"
 
-extern ConfigGroup db_backend_config;
-extern ConfigVariable<std::string> db_backend_type;
+
+#if NEW_BACKEND_CONFIG
```

Comment:
Why do we have this in here as an if? If we're switching over the system, we should remove the old capability entirely.